rfcs/text/0003-object-field.md

```diff
@@ -79,7 +88,7 @@ The goal here is to research and understand the impact of these changes on users
 <!--
 Stage 1: Identify potential concerns, implementation challenges, or complexity. Spend some time on this. Play devil's advocate. Try to identify the sort of non-obvious challenges that tend to surface later. The goal here is to surface risks early, allow everyone the time to work through them, and ultimately document resolution for posterity's sake.
 -->
-An object is a very broad categorization, in some situations it may be hard to determine when something should be normalized into this field or another field like `file` or `package`. Additionally given the genericness of this field, a lot of finer details about the object may not be represented by this field, but rather addressed in custom key/value pairs nested under this field. Also due to the genericness of the field, the `object.type` field becomes rather critical to designate what exactly is represented within this field (e.g. is it a `record` or a `meeting`); some context might be derived from other fields in the event, but that would require some foreknowledge in order to properly search or sort. Designating specific acceptable values for `object.type` could minimize any confusion around the object being represented; however that would require specific updates to ECS to support new object "types".
+An `object` - as initially proposed - is a very broad categorization, in some situations it may be hard to determine when something should be normalized into this field or another field like `file` or `package`. the **Enterprise Content** terminology seeks to address this by focusing on the non-time series nature of the target documents. Additionally given the genericness of this field, a lot of finer details about the object may not be represented by this field, but rather addressed in custom key/value pairs nested under this field. `object.type` conventions will be critical to designate what exactly is represented within this field (e.g. is it a `record` or a `meeting`); some context might be derived from other fields in the event, but that would require some foreknowledge in order to properly search, sort or infer. Designating specific acceptable values for `object.type` could minimize any confusion around the object being represented; however that would require specific updates to ECS to support new object "types".
```

Comment:
```suggestion
An `object` - as initially proposed - is a very broad categorization. In some situations it may be hard to determine when something should be normalized into this field or another field like `file` or `package`. The **Enterprise Content** terminology seeks to address this by focusing on the non-time series nature of the target documents. Additionally given the genericness of this field, a lot of finer details about the object may not be represented by this field, but rather addressed in custom key/value pairs nested under this field. `object.type` conventions will be critical to designate what exactly is represented within this field (e.g. is it a `record` or a `meeting`); some context might be derived from other fields in the event, but that would require some foreknowledge in order to properly search, sort or infer. Designating specific acceptable values for `object.type` could minimize any confusion around the object being represented; however that would require specific updates to ECS to support new object "types".
```

Diff isn't very useful here. Just a couple of minor proofreading items noted while reviewing.

Reply:
Thank you